.gitignore

```diff
@@ -10,3 +10,6 @@
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
+
```

Comment:
avoid including `vendor/` code in the repo, makes life a lot simpler in the long run.

Reply:
Fair, I was following the practices of the private internal repositories

Reply:
Yeah fair enough. We include vendor'd deps in the internal repos for build reproducibility, so fortunately we can avoid including them upstream (GH/etc)